src/Components/FormCreation/CardComponents/SelectQuestion.jsx

```diff
@@ -102,7 +101,6 @@ function SelectQuestion({
     event.target.blur();
   };
 
-  console.log(options);
   const onEditOption = (index, value) => {
     console.log("Editing Value");
```

Comment:
Not in this PR, but there appears to be at least 1 more log in this file. If you want, I think that would be great to filter.